spring-webflux-threads/pom.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.baeldung.spring</groupId>
+    <artifactId>spring-webflux-threads</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>spring-webflux-threads</name>
+    <packaging>jar</packaging>
+    <description>Spring WebFlux AMQP Sample</description>
+
+    <parent>
+        <groupId>com.baeldung</groupId>
+        <artifactId>parent-boot-2</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+        <relativePath>../parent-boot-2</relativePath>
+    </parent>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-webflux</artifactId>
+            <!--
```

Comment:
Delete commented out lines. The same happens below

Reply:
The commented lines are actually a way to suggest how to change the default runtime.

Reply:
I have added comments in the pom to indicate why the lines are commented.